tests/test_feature.rb

```diff
@@ -133,6 +133,10 @@ def test_pim
     feature('pim')
   end
 
+  def test_private_vlan
+    feature('private_vlan')
```

Comment:
@davidefdl I discovered a bug in this file.  Would you mind including the fix in this PR?  Simple fix.
```ruby
diff --git a/tests/test_feature.rb b/tests/test_feature.rb
index 7223b98..f5a438f 100755
--- a/tests/test_feature.rb
+++ b/tests/test_feature.rb
@@ -51,7 +51,7 @@ class TestFeature < CiscoTestCase
 
     # Get current state of feature, then disable it
     pre_clean_enabled = Feature.send("#{feat}_enabled?")
-    config("no #{feat_str}") if pre_clean_enabled
+    config("no feature #{feat_str}") if pre_clean_enabled
     refute_show_match(
       command: "show running | i #{feat_str}",
       pattern: /^#{feat_str}$/,
@@ -68,7 +68,7 @@ class TestFeature < CiscoTestCase
            "Feature #{feat} (#{feat_str}) is not enabled")
 
     # Return testbed to pre-clean state
-    config("no #{feat_str}") unless pre_clean_enabled
+    config("no feature #{feat_str}") unless pre_clean_enabled
   end
 
   ###################
```
The tests were trying remove the features without the `feature` keyword.